### ISSUE TITLE
RccpExports always gives Trailing blank line

### ISSUE THREAD

**Issue by EdwinTh** [jimhester/lintr#282]
Firts of all my n = 1, since I have one package in which Rcpp is used. 

The file RcppExports.R that is createad by Rcpp always produces a Trailing blank line. I have to remove it at every rebuild to be lint free.

If this is consistent, IMO it might be desirable to ignore RcppExports when linting. Otherwise you have to remove the trailing line in this file after each build to be lint free.

**Comment by kenahoo** [comment]
I came here to report the same problem.  Options are:

1. Fix the problem upstream in `Rcpp::compileAttributes()`.  Not sure they should really have to worry about lint checks in their generated code, though.
2. Always ignore `R/RcppExports.R` when linting - seems reasonable, but also kind of special-casey and hard-codey.
3. Detect auto-generated files by some regex on the contents of the file - most of them have something like Rcpp's `"Generated by using Rcpp::compileAttributes() -> do not edit by hand"` comment.  Probably not very standard, though.
4. Ask people to add `exclude="R/RcppExports.R"` to their `lintr::expect_lint_free()` call in the tests - this doesn't seem to actually exclude the file, and it also causes any lines I've marked as `# nolint` to fail with whatever their linting issue is.  And it seems sub-ideal to ask everyone who uses Rcpp to do this.
